### ISSUE TITLE
Progressive Web App

### ISSUE THREAD

**Issue by MakingMadness** [anapnoe/stable-diffusion-webui-ux#13]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Would you consider configuring the webui as a PWA? It would allow for fast switching between it and other apps, making more complex workflows easier to do on Android devices.

### Proposed workflow

1. Go to Firefox menu on Android
2. Press Install.
3. Use website as app.

**Comment by anapnoe** [comment]
Yes I am considering it but it will take some time to focus on this
